src/core/components/layout/stories/tiles/collapse-below.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { Tiles } from "../../index"
+import { sport } from "@guardian/src-foundations/palette"
+import { css } from "@emotion/core"
+
+const contents = css`
+	text-align: center;
+	background-color: ${sport[600]};
+`
```

Comment:
Can we use `textSans.medium()` here, for consistency with the [default stories](https://github.com/guardian/source/blob/56d2501bcbebeeb12dcec83c93b1d6a1289ba45e/src/core/components/layout/stories/tiles/default.tsx#L8)?